src/App.tsx

```diff
@@ -1,27 +1,109 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import 'bulma/css/bulma.min.css';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { Good } from './types/Good';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import {
+  getAll,
+  get5First,
+  getRedGoods,
+  wait,
+} from './api/goods';
+import { LoadingError } from './components/LoadingError';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [loadingFirstFive, setLoadingFirstFive] = useState(false);
+  const [loadingRed, setLoadingRed] = useState(false);
+  const [hasError, setHasError] = useState(false);
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+  const handleAllGoods = async () => {
+    setLoading(true);
+    setHasError(false);
+    await wait(1000);
```

Comment:
Why do we need to increase waiting time, we already make real requests that take time for loading